Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Scaffolding/Internal/FbDatabaseModelFactory.cs

```diff
@@ -163,12 +163,18 @@ CASE Coalesce(F.RDB$FIELD_TYPE, 0)
 				 WHEN 0 THEN 'BIGINT'
 				 ELSE 'DECIMAL'
 				END
+                WHEN 23 THEN 'BOOLEAN'
```

Comment:
Nit: Indentation.

Reply:
Still see different indentation. Maybe the original code has wrong one. If so, just let me know and I'll fix it first.

Reply:
Interesting. If i look at the file as plain text, the indentations seem to be fine. Maybe its an issue in the Github viewer?
![image](https://user-images.githubusercontent.com/67638665/114730436-28549900-9d41-11eb-8b87-a15377f96054.png)
Not sure whats happening here